lib/completions/endpoints/fake.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module Completions
+    module Endpoints
+      class Fake < Base
```

Comment:
Like this idea! We have a `CannedResponse` object which we use for tests. Maybe we could adapt it for this use case?

Reply:
Yeah possibly, the use case here was talking to "fake" bot. but it would be nice to give it a series of canned responses. 

eg: if you see "table" in the request you make a markdown table ... and so on.